test/jdk/java/awt/regtesthelpers/Util.java

```diff
@@ -174,6 +174,8 @@ public static boolean testPixelColor(int x, int y, final Color color, int attemp
             if (screen.equals(color)) {
                 return true;
             }
+            System.out.println("Robot.getPixelColor Expected: " + color);
+            System.out.println("Robot.getPixelColor Actual: " + screen);
```

Comment:
This may be not the best solution. If the method is used on a large image / rectangle (it's called from `testBoundsColor`), the output could be flooded with messages. Eventually, it may slow down test execution. At the same time, there's no other way to provide debugging info.